src/nbody/nbody.cpp

```diff
@@ -84,7 +87,7 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   Real Rf[3] = {0.0};
   Real Vf[3] = {0.0};
   if (global_frame && (Omf != 0.0) && (qshear != 0.0)) {
-    const Real R0 = std::pow(SQR(Omf) / GM, 1.0 / 3.0);
+    const Real R0 = std::pow(SQR(Omf) / units.GetGCode(), 1.0 / 3.0);
```

Comment:
This should really be GM not G

Reply:
The `M` here is the sum of the particles' `p.m` values right? So should this value be set in `NBodySetup` as `units.GetGCode() * mtot`?

Reply:
Since `GM` is really set at startup and put into params, I don't think we need any units stuff in places where it's used. When it's initially defined it should probably just be `GM = units.GetGCode() * units.GetMCode() * GM`.

Reply:
discussion moved below, resolving this